website/docs/reference/appsmith-framework/widget-actions/store-value.md

```diff
@@ -188,6 +188,22 @@ If you want to clear the session store value, you must define three parameters a
 }}
 ```
 
-### What's Next
+
+> Your app may face performance issues if you make multiple calls to the ```storeValue``` function to store values. It is **recommended** to use an array object to store values. For instance: 
```

Comment:
I think a heading would be good for this section. Right now, it looks like a part of clear session store value.

Reply:
On second thoughts, @danciaclara - Could this be moved to the troubleshooting guide? Do we know what error message is shown to the user. The store value doc can cross-reference the troubleshooting guide. Your thoughts??

Reply:
My initial suggestion to the engineering team was that, but this is a performance issue (no error messages. app just slows down), so sort of a like best practice of using store value in this scenario. So let's add it in the doc itself

@harshilp24 let's not explicitly say that users will face performance issues :) just mention something like 

> If you need to store many values, instead of making multiple calls to the ```storeValue``` function, it is **recommended** to use an object array to store the values. For instance:

Reply:
The heading can be **Storing multiple values**

Reply:
Should I include this inside the tip (green disclaimer) or add best practice title @danciaclara @jnikhila

Reply:
@harshilp24 no, this is not a callout, just keep it a subheading "Storing multiple values"

Reply:
done